src/main/webapp/creatives/maps/simid_maps.js

```diff
@@ -5,4 +5,25 @@ class SimidMapCreative extends BaseSimidCreative {
   constructor() {
     super();
   }
+  /** @override */
+  onStart(eventData){
+    super.onStart(eventData);
+    this.adParams = JSON.parse(this.creativeData.adParameters);
+    console.log("Ad params: ", this.adParams);
+    var paramObject = this.adParams[0];
+    console.log(paramObject);
+    var buttonParam = paramObject["buttonParam"];
```

Comment:
"buttonParam" and any other params you use like that can be top-level consts

Reply:
fixed